README.md

```diff
@@ -128,6 +128,42 @@ async def modify_resources(resources):
 
 Modifiers are called sequentially in the order in which they were registered.
 
+### Callbacks
+
+Callbacks are external function from your own application that can be executed at specific times during the `tc-admin apply` execution:
+
+* A `before_apply` callback will run before a resource is created, updated or deleted,
+* A `after_apply` callback will run after a resource has beencreated, updated or deleted.
+
+Supported actions are :
+
+* create
+* update
+* delete
+
+By default all actions are used.
+
+Supported resources are:
+* `tcadmin.resources.Role`
+* `tcadmin.resources.Hook`
+* `tcadmin.resources.WorkerPool`
+* `tcadmin.resources.Client`
+* `tcadmin.resources.Secret`
```

Comment:
I think this is all resources.  Would it be easier to just say that, rather than remember to update this list if/when we add a new resource type?